gui/src/components/health/cortx-severity-based-health-view.vue

```diff
@@ -16,15 +16,16 @@
 */
 <template>
   <div class="pa-5">
-    <div
-      class="cortx-back-btn"
-      @click="$router.go(-1)"
-    >
+    <div class="cortx-back-btn" @click="$router.go(-1)">
```

Comment:
why do we require any change in this file?

Reply:
Resolved